orgs-using-d.dd

```diff
@@ -0,0 +1,123 @@
+Ddoc
+
+$(D_S $(TITLE),
+
+$(COMMENT
+    - Put your logo (png/svg) to images/orgs-using-d
+    - max. dims: 200x100 (resize with e.g. `mogrify -resize 200x100 image.png`)
+    - if you need commas, use `$(COMMA`)
+)
+$(DIVC orgs-using-d center,
+
+    $(DORG Facebook, https://www.facebook.com, facebook.svg,
+        Online social networking service,
+        $(HTTPS https://github.com/facebookarchive/warp, C Preprocessor warp) and $(HTTP www.drdobbs.com/mobile/facebook-adopts-d-language/240162694, more infrastructure tools).
+        $(LINK_ROW
+            $(FA_TALK http://dconf.org/2014/talks/simpkins.html, DConf talk)
```

Comment:
`FA_TALK` forwards to `WEB` which forwards to `HTTP` which prepends `http://`. So no `http://` here, please.